test/common/gitops_utils.go

```diff
@@ -165,4 +165,13 @@ func GitOpsCleanup(user OCPUser) {
 	if !k8serrors.IsNotFound(err) {
 		ExpectWithOffset(1, err).ShouldNot(HaveOccurred())
 	}
+
+	gitopsTestNamespaces := []string{
```

Comment:
Can this be a package-level variable rather than redefining it?

Reply:
Good point--I'll adjust it.